packages/fast-usdc/test/utils/fees.test.ts

```diff
@@ -161,6 +170,42 @@ test(feeToolsScenario, {
   },
 });
 
+test(feeToolsScenario, {
+  name: 'AGORIC_PLUS_ETHEREUM with commonPrivateArgs.feeConfig',
+  // 950_000_000n
+  requested: USDC(MockCctpTxEvidences.AGORIC_PLUS_ETHEREUM().tx.amount),
+  config: makeTestFeeConfig(withAmountUtils(issuerKits.USDC)),
+  expected: {
+    totalFee: USDC(19000001n), // 1n + 2% of 950USDC
+    advance: USDC(930999999n),
+    split: {
+      ContractFee: USDC(3800000n), // 20% of fee
+      PoolFee: USDC(15200001n),
+    },
+  },
+});
+
+test(feeToolsScenario, {
+  name: 'AGORIC_PLUS_ETHEREUM with customFeeConfigs',
+  requested: USDC(MockCctpTxEvidences.AGORIC_PLUS_ETHEREUM().tx.amount),
+  destination: 'eip155:1:0x1234567890123456789012345678901234567890',
+  config: makeTestFeeConfig(withAmountUtils(issuerKits.USDC)),
+  customFeeConfigs: harden({
+    'eip155:1': {
+      ...makeTestFeeConfig(withAmountUtils(issuerKits.USDC)),
+      flat: USDC(500_000n),
+    },
+  }),
```

Comment:
I added [a note in #11149](https://github.com/Agoric/agoric-sdk/issues/11149#issuecomment-2777194621) about testing product's specific choice of fees too.

Reply:
Thank you